layouts/shortcodes/deploykf/OWNERS

```diff
@@ -0,0 +1,2 @@
+approvers:
+  - ca-scribner
```

Comment:
This one might be a copy-paste error - I don't think it is me :)

Reply:
Yea, it should be `thesuperzapper`, lol:

```suggestion
  - thesuperzapper
```

Reply:
Nice catch, let me fix it.